restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/telephony/CallLifecycleTest.java

```diff
@@ -288,6 +289,13 @@ public void testDialCancelBeforeDialingClientAliceAfterTrying() throws ParseExce
         assertTrue(maxConcurrentCalls==0);
         assertTrue(maxConcurrentIncomingCalls==0);
         assertTrue(maxConcurrentOutgoingCalls==0);
+
+        Map<String, Integer> mgcpResources = MonitoringServiceTool.getInstance().getMgcpResources(metrics);
```

Comment:
this will be a fairly common construct, could we isolate this in a Assert util class, where access to monitoring services, and literal to specific counters are protected

Reply:
@jaimecasero I thought about that but let this task for later as it will take significant time. I did half of the work (`MonitoringServiceTool.getInstance().getMgcpResources(metrics)`) but the rest of the enhancement can be done in another sprint with proper estimation

Reply:
ok